### PR TITLE
Add spelling corrections for evaluate

### PR DESCRIPTION
**Pull request by adrien-berchet**
https://github.com/search?q=evaliate&type=code => 548 hits
https://github.com/search?q=evaliation&type=code => 788 hits
https://github.com/search?q=evaliated&type=code => 45 hits
https://github.com/search?q=evaliates&type=code => 10hits
https://github.com/search?q=evaliating&type=code => 141 hits
https://github.com/search?q=evaliations&type=code => 61 hits

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -22818,6 +22818,8 @@ Eurpean->European
 Eurpoean->European
 evailable->available
 evalation->evaluation
+evaliate->evaluate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:22821]: May I suggest all verbal forms (`evaliated`, `evaliates`, `evaliating`)?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -22818,6 +22818,8 @@ Eurpean->European
...
+evaliation->evaluation
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:22822]: Plural `evaliations` too?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:22821]: Can you find any typos of `validate`/`revalidate`?

Comment (adrien-berchet) [reply codespell_lib/data/dictionary.txt:22821]: Sure!

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@adrien-berchet Thank you for the contribution, much appreciated.

**Pull request merged**